src/main/webapp/index.html

```diff
@@ -60,7 +60,7 @@ <h3 href="products.html" class="brand-name">Style Advisor</h3>
                 <div class="heading">
                     <img src="images/home/new-profile.svg" class="logo"> <!-- Style Advisor Logo (coming soon) -->
                     <h2 id="welcome">Welcome</h2>
-                    <h4>Sign in to your account</h4>
+                    <!--<h4>Sign in to your account</h4>
```

Comment:
Why not just delete it, instead of commenting?

Same comment for below changes.